config/config.js

```diff
@@ -28,8 +28,32 @@ const config = {
         database: 'test'
     },
 
+    VALIDATOR: {
+        /**
+         * Add custom validators here
+         * Format: { "validatorName": function(value, [additional arguments]), ... }
+         */
+        customValidators: {
```

Comment:
I don't think we check the types with these custom validators though. Sorry to going back & forth but I think we want to have 
1. define the input as a schema and validate
2. get the data in actual parsed/cleaned types (future milestone?)

In the meantime, maybe
1. switch back to the larval port since it has quite a few built schema format built in
2. add our custom filter by leveraging https://www.npmjs.com/package/validator (we can add all the validators to our own customValidators by adding all the isXXXX function by adding all the functions and remove the is in the function name
3. build our own custom filter for data types etc
4. look and compare with another library 
https://www.npmjs.com/package/express-jsonschema